crash on Android Q

I test the library targeting Android Q
I have a crash. Seems related to reflection 
`     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object java.lang.reflect.Field.get(java.lang.Object)' on a null object reference
        at io.github.inflationx.viewpump.ReflectionUtils.getValue(ReflectionUtils.java:25)
        at io.github.inflationx.viewpump.ViewPumpLayoutInflater.createCustomViewInternal(ViewPumpLayoutInflater.java:196)
        at io.github.inflationx.viewpump.ViewPumpLayoutInflater.access$000(ViewPumpLayoutInflater.java:14)
        at io.github.inflationx.viewpump.ViewPumpLayoutInflater$PrivateWrapperFactory2ViewCreator.onCreateView(ViewPumpLayoutInflater.java:405)
        at io.github.inflationx.viewpump.FallbackViewCreationInterceptor.intercept(FallbackViewCreationInterceptor.java:11)
        at io.github.inflationx.viewpump.InterceptorChain.proceed(InterceptorChain.java:37)
        at io.github.inflationx.calligraphy3.CalligraphyInterceptor.intercept(CalligraphyInterceptor.java:18)
        at io.github.inflationx.viewpump.InterceptorChain.proceed(InterceptorChain.java:37)
        at io.github.inflationx.viewpump.ViewPump.inflate(ViewPump.java:49)
        at io.github.inflationx.viewpump.ViewPumpLayoutInflater$PrivateWrapperFactory2.onCreateView(ViewPumpLayoutInflater.java:385)
        at android.view.LayoutInflater$FactoryMerger.onCreateView(LayoutInflater.java:235)`

--- a/calligraphy/src/main/java/io/github/inflationx/calligraphy3/Calligraphy.java
+++ b/calligraphy/src/main/java/io/github/inflationx/calligraphy3/Calligraphy.java
@@ -15,7 +15,6 @@ import java.lang.ref.WeakReference;
 import java.lang.reflect.Method;
 
 import androidx.appcompat.widget.Toolbar;
-import io.github.inflationx.viewpump.ReflectionUtils;
 
 class Calligraphy {
 

--- a/calligraphy/src/main/java/io/github/inflationx/calligraphy3/ReflectionUtils.java
+++ b/calligraphy/src/main/java/io/github/inflationx/calligraphy3/ReflectionUtils.java
@@ -1,0 +1,34 @@
+package io.github.inflationx.calligraphy3;
+
+import android.util.Log;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+class ReflectionUtils {
+
+    private static final String TAG = ReflectionUtils.class.getSimpleName();
+
+    static Method getMethod(Class clazz, String methodName) {
+        final Method[] methods = clazz.getMethods();
+        for (Method method : methods) {
+            if (method.getName().equals(methodName)) {
+                method.setAccessible(true);
+                return method;
+            }
+        }
+        return null;
+    }
+
+    static void invokeMethod(Object object, Method method, Object... args) {
+        try {
+            if (method == null) return;
+            method.invoke(object, args);
+        } catch (IllegalAccessException e) {
+            Log.d(TAG, "Can't access method using reflection", e);
+        } catch (InvocationTargetException e) {
+            Log.d(TAG, "Can't invoke method using reflection", e);
+        }
+    }
+}

